test/unit/main.cpp

```diff
@@ -1,2 +1,10 @@
 #define CATCH_CONFIG_MAIN
 #include <catch2/catch_all.hpp>
+
+#include <uenv/log.h>
+
+static struct unit_init_log {
+    unit_init_log() {
+        uenv::init_log(spdlog::level::trace, spdlog::level::off);
```

Comment:
Logs from tests should show up now, you just have to make sure the messages use an appropriate level.

I now disabled the syslog output for unit tests because I figured that's not useful and just ends up spamming syslog.

I made the default stderr logging use `trace`, since they're unit tests and probably doesn't hurt to be more verbose there? It depends a bit on how much logging you're planning to use. If we set this to something other than `trace` it might be nice to e.g. allow changing the level with `UENV_UNIT_TEST_LOG_LEVEL` or similar, but this we can also add when/if the need arises.